scripts/install_deps.sh

```diff
@@ -18,22 +18,40 @@ curl_flags=(
 run_trace false sudo apt-get install --no-install-recommends --yes \
   shellcheck
 
-run_trace false curl "${curl_flags[@]}" "https://get.helm.sh/helm-v3.7.1-linux-amd64.tar.gz" \| \
-  tar -C "$TMPDIR" --strip-components=1 -zxf - linux-amd64/helm
+# gotestsum makes test output more readable
+GOTESTSUM_VERSION=1.7.0
```

Comment:
nit: (non-blocking): I find that puting `XXX_VERSION` at the very top of the file, all clustered together, makes it somewhat easier. My personal preference though entirely.

Reply:
I can see that, yeah - I guess the downside is that separating them makes it easier to remove code without removing corresponding env vars :thinking: I think I'll leave it as-is for now, but I also don't feel strongly about it and might change my mind later lol

I actually want to move all of these to an image build (and build our stuff inside an image), so this script is kinda throwaway code